src/config/configVault.ts

```diff
@@ -63,6 +65,10 @@ class ConfigVault {
     this.store.set('configs', newConfigs)
   }
 
+  private async setActiveProjectSummary(projectId: string): Promise<void> {
```

Comment:
Since you've changed the logic of the method (which I quite like), let's rename it
```suggestion
  private async updateActiveProjectSummary(projectId: string): Promise<void> {
```